tools/Ensembl-REST/get_feature_info.xml

```diff
@@ -36,7 +36,13 @@ python '$__tool_directory__/get_feature_info.py'
         <test>
             <param name="input" ftype="txt" value="input.txt" />
             <param name="expand" value="false" />
-            <output name="output" file="out.json" />
+            <output name="output" file="out.json">
```

Comment:
I think you need to remove `file` and `ftype` attributes when you use `assert_contents`